kover-offline-runtime/src/main/java/kotlinx/kover/offline/runtime/api/KoverRuntime.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.offline.runtime.api;
+
+import com.intellij.rt.coverage.offline.api.CoverageRuntime;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Utility Kover to collect coverage inside JVM process in which the tests were run.
+ */ 
+public class KoverRuntime {
+
+    /**
+     * Get classes coverage. For the correct collecting of coverage, an analysis of the class-files is required.
+     * <p/>
+     * Calling this method is allowed only after all tests are completed. If the method is called in parallel with the execution of the measured code, the coverage value is unpredictable.
+     *
+     * @param classFileRoots root directories containing non-instrumented class-files the coverage of which
+     *                       needs to be measured, or inline functions of the measured classes are called in them.
```

Comment:
Not clear what 'or inline functions of the measured classes are called in them' means, better to split it into separate sentences

Reply:
No longer actual